policyengine_us/parameters/gov/states/co/tax/income/credits/sales_tax_refund/amount/base.yaml

```diff
@@ -17,33 +19,44 @@ brackets:
     amount:
       2021-01-01: 37
       2022-01-01: 153
+      2023-01-01: 454
```

Comment:
please use the amounts with the new Bill: it was signed into law and the amounts increased

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/251e6068-b530-4baa-be4e-ac2e05e8c5ed)

Reply:
the second column is the amount that is applicable not the first